specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -808,6 +808,65 @@
         }
       }
     },
+    "/keys/{key-name}/{key-version}/export": {
+      "post": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "exportKey",
+        "summary": "Exports a key.",
+        "description": "The export key operation is applicable to all key types. The target key must be marked exportable. This operation requires the keys/export permission.",
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key to get."
+          },
+          {
+            "name": "key-version",
+            "in": "path",
+            "required": true,
```

Comment:
I don't believe this is actually required. It's not for other artifact operations.

Reply:
This is checked in code from 7.2 I'll pass this comment info to the original authors.